### ISSUE TITLE
Toggle Switch, Checkbox - adjust component documentation

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/boiler#1071]
### Description / User story

The "ariaLabel" prop is not placed correctly in the props panel of the Toggle Switch, it should be placed in the section "Accessibility" right after the "States" section. Also the prop has the following description: "Provides additional   information about the elements purpose and functionality to assistive technologies, such as screen readers."

The "indeterminatedIcon" prop has been renamed to "indeterminateIcon" for the Checkbox (also in the props panel).

### Requirements / Prerequisites

- none

### Acceptance Criteria

- [ ] Acceptance Criteria 1
- [ ] ...

### Additional information

- ...

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
